rollup.config.ts

```diff
@@ -0,0 +1,44 @@
+import replace from '@rollup/plugin-replace'
+import { defineConfig } from 'rollup'
+import rename from 'rollup-plugin-rename'
+import typescript from 'rollup-plugin-ts'
```

Comment:
I would propose to use esbuild instead of rollup-plugin-ts is much faster

have a look at rollup-plugin-esbuild for that

Reply:
I just needed something that would resolve the `tsconfig.json` file, I'll take a look at it in the morning :+1:

Reply:
Never mind rollup-pluigin-ts is better than I thought leave it in! No change required (as type-files get worse when useing esbuild)!